client/src/constants/participantTableConstants.js

```diff
@@ -98,3 +67,217 @@ export const defaultTableState = {
   locationFilter: '',
   siteSelector: '',
 };
+
+export const tabStatuses = {
+  'Available Participants': ['open'],
+  'My Candidates': ['prospecting', 'interviewing', 'offer_made', 'unavailable'],
+  'Archived Candidates': ['rejected'],
+  'Hired Candidates': ['hired'],
+  Participants: ['open', 'prospecting', 'interviewing', 'offer_made', 'rejected', 'hired'],
+};
+
+export const tabsByRole = {
+  superuser: ['Participants'],
+  ministry_of_health: ['Participants'],
+  health_authority: [
+    'Available Participants',
+    'My Candidates',
+    'Archived Candidates',
+    'Hired Candidates',
+  ],
+  employer: ['Available Participants', 'My Candidates', 'Archived Candidates', 'Hired Candidates'],
+};
+
+const columns = {
+  id: { id: 'id', name: 'ID', sortOrder: 1 },
+  lastName: { id: 'lastName', name: 'Last Name', sortOrder: 2 },
+  firstName: { id: 'firstName', name: 'First Name', sortOrder: 3 },
+  status: { id: 'status', name: 'Status', sortOrder: 4 },
+  statusInfo: { id: 'statusInfo', name: 'Status', sortOrder: 5 },
+  postalCodeFsa: { id: 'postalCodeFsa', name: 'FSA', sortOrder: 6 },
+  phoneNumber: { id: 'phoneNumber', name: 'Phone Number', sortOrder: 7 },
+  emailAddress: { id: 'emailAddress', name: 'Email Address', sortOrder: 8 },
+  preferredLocation: { id: 'preferredLocation', name: 'Preferred Region(s)', sortOrder: 9 },
+  distance: { id: 'distance', name: 'Site Distance', sortOrder: 10 },
+  interested: { id: 'interested', name: 'Interest', sortOrder: 11 },
+  nonHCAP: { id: 'nonHCAP', name: 'Non-HCAP', sortOrder: 12 },
+  crcClear: { id: 'crcClear', name: 'CRC Clear', sortOrder: 13 },
+  callbackStatus: { id: 'callbackStatus', name: 'Callback Status', sortOrder: 14 },
+  userUpdatedAt: { id: 'userUpdatedAt', name: 'Last Updated', sortOrder: 15 },
+  engage: { id: 'engage', name: null, sortOrder: 16 },
+  edit: { id: 'edit', name: null, sortOrder: 17 },
+  siteName: { id: 'siteName', name: 'Site Name', sortOrder: 18 },
```

Comment:
Looks like you'll need an 'Archive' column here. 

`archive: {id:'archive',name:null, sortOrder:19}`